zellij-server/src/panes/plugin_pane.rs

```diff
@@ -251,10 +251,15 @@ impl Pane for PluginPane {
                 pane_title,
                 frame_params,
             );
-            Some(frame.render())
+            Some(
+                frame
+                    .render()
+                    .with_context(|| format!("failed to render frame form client {_client_id}"))?,
```

Comment:
There's a little typo here:
```suggestion
                    .with_context(|| format!("failed to render frame for client {_client_id}"))?,
```